app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post
+  after_save :update_the_post_counter
+
+  def update_the_comment_counter
```

Comment:
**Great implementation based on the previous reviewer's request.** 

[OPTIONAL] Most of the time if we use the method with `after_save` or `before_save`... we put the used method after `private` to make it accessible only in this class. So, if you are using a method like this. I suggest you put it after `private`, or if you want to use this in other classes, feel free to keep it like that :)